src/main/java/Menu.java

```diff
@@ -0,0 +1,37 @@
+class Menu {
+    public static void main(String[] args) {
+        Drink drink1 = new Drink();
+        drink1.name ="Mojito";
+        drink1.price = 13.0;
+        drink1.alcoholic = true;
+        drink1.ingridients = new Ingridients();
+        drink1.ingridients.ing1 = "woda";
+        drink1.ingridients.ing2 = "sok";
+        drink1.ingridients.ing3 = "wódka";
+        drink1.ingridients.ing1Vol = 150;
+        drink1.ingridients.ing2Vol = 110;
+        drink1.ingridients.ing3Vol = 50;
+        drink1.drinkVol = (drink1.ingridients.ing1Vol + drink1.ingridients.ing2Vol + drink1.ingridients.ing3Vol);
+
+        Drink drink2 = new Drink();
+        drink2.name ="Caipirinha";
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu